ts/src/config.ts

```diff
@@ -106,5 +129,8 @@ export const defaultConfig = {
   timeIntervalMicros: 1000,
   heapIntervalBytes: 512 * 1024,
   heapMaxStackDepth: 64,
-  backoffMillis: 5 * 60 * 1000
+  initialBackoffMillis: 1000,
+  expBackoffMillisCap: parseDuration('1h'),
+  backoffMultiplier: 1.3,
+  serverBackoffMillisCap: parseDuration('7d'),  // 7 days
```

Comment:
The "// 7 days" comment is now redundant.

Reply:
Done.